[allegro5] Build error on x64-linux

Hello, I got this issue when just called the command "vcpkg install allegro5".
Thank you in advance!

Package: allegro5:x64-linux@5.2.9.0

**Host Environment**

- Host: x64-linux
- Compiler: GNU 13.2.1
-    vcpkg-tool version: 2023-12-12-1c9ec1978a6b0c2b39c9e9554a96e3e275f7556e
    vcpkg-scripts version: f1c6efee2 2023-12-24 (27 hours ago)

**To Reproduce**

`vcpkg install allegro5`
**Failure logs**

```
-- Using cached liballeg-allegro5-5.2.9.0.tar.gz.
-- Cleaning sources at /home/p_matveev/Program/VenGame/vcpkg/buildtrees/allegro5/src/5.2.9.0-9723edc001.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/p_matveev/Program/VenGame/vcpkg/downloads/liballeg-allegro5-5.2.9.0.tar.gz
-- Applying patch do-not-copy-pdbs-to-lib.patch
-- Applying patch msvc-arm64-atomic.patch
-- Using source at /home/p_matveev/Program/VenGame/vcpkg/buildtrees/allegro5/src/5.2.9.0-9723edc001.clean
-- Found external ninja('1.11.1').
-- Configuring x64-linux
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: /usr/bin/ninja -v
    Working Directory: /home/p_matveev/Program/VenGame/vcpkg/buildtrees/allegro5/x64-linux-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      /home/p_matveev/Program/VenGame/vcpkg/buildtrees/allegro5/config-x64-linux-dbg-CMakeCache.txt.log
      /home/p_matveev/Program/VenGame/vcpkg/buildtrees/allegro5/config-x64-linux-rel-CMakeCache.txt.log
      /home/p_matveev/Program/VenGame/vcpkg/buildtrees/allegro5/config-x64-linux-out.log

Call Stack (most recent call first):
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:252 (vcpkg_execute_required_process)
  ports/allegro5/portfile.cmake:15 (vcpkg_cmake_configure)
  scripts/ports.cmake:170 (include)



```
<details><summary>/home/p_matveev/Program/VenGame/vcpkg/buildtrees/allegro5/config-x64-linux-out.log</summary>

```
[1/2] "/usr/bin/cmake" -E chdir ".." "/usr/bin/cmake" "/home/p_matveev/Program/VenGame/vcpkg/buildtrees/allegro5/src/5.2.9.0-9723edc001.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=/home/p_matveev/Program/VenGame/vcpkg/packages/allegro5_x64-linux" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DCMAKE_PROJECT_INCLUDE=/home/p_matveev/Program/VenGame/vcpkg/ports/allegro5/cmake-project-include.cmake" "-DWANT_DOCS=OFF" "-DALLEGRO_SDL=OFF" "-DWANT_DEMO=OFF" "-DSHARED=0" "-DINSTALL_PKG_CONFIG_FILES=true" "-DWANT_EXAMPLES=OFF" "-DWANT_TESTS=OFF" "-DWANT_AUDIO=ON" "-DWANT_COLOR=ON" "-DWANT_D3D=ON" "-DWANT_D3D9EX=OFF" "-DWANT_DSOUND=ON" "-DWANT_FLAC=ON" "-DWANT_FONT=ON" "-DWANT_GLES3=ON" "-DWANT_IMAGE=ON" "-DWANT_IMAGE_JPG=ON" "-DWANT_IMAGE_PNG=ON" "-DWANT_MEMFILE=ON" "-DWANT_MODAUDIO=OFF" "-DWANT_NATIVE_DIALOG=ON" "-DWANT_NATIVE_IMAGE_LOADER=ON" "-DWANT_OGG_VIDEO=ON" "-DWANT_OPENAL=ON" "-DWANT_OPENGL=ON" "-DWANT_OPENSL=OFF" "-DWANT_OPUS=ON" "-DWANT_PHYSFS=ON" "-DWANT_POPUP_EXAMPLES=OFF" "-DWANT_PRIMITIVES=ON" "-DWANT_RELEASE_LOGGING=OFF" "-DWANT_SHADERS_D3D=ON" "-DWANT_SHADERS_GL=ON" "-DWANT_TREMOR=OFF" "-DWANT_TTF=ON" "-DWANT_VIDEO=ON" "-DWANT_VORBIS=ON" "-DCMAKE_MAKE_PROGRAM=/usr/bin/ninja" "-DCMAKE_SYSTEM_NAME=Linux" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/home/p_matveev/Program/VenGame/vcpkg/scripts/toolchains/linux.cmake" "-DVCPKG_TARGET_TRIPLET=x64-linux" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/home/p_matveev/Program/VenGame/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/home/p_matveev/Program/VenGame/vcpkg" "-D_VCPKG_INSTALLED_DIR=/home/p_matveev/Program/VenGame/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DWANT_ALLOW_SSE=ON"
FAILED: ../CMakeCache.txt 
"/usr/bin/cmake" -E chdir ".." "/usr/bin/cmake" "/home/p_matveev/Program/VenGame/vcpkg/buildtrees/allegro5/src/5.2.9.0-9723edc001.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=/home/p_matveev/Program/VenGame/vcpkg/packages/allegro5_x64-linux" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DCMAKE_PROJECT_INCLUDE=/home/p_matveev/Program/VenGame/vcpkg/ports/allegro5/cmake-project-include.cmake" "-DWANT_DOCS=OFF" "-DALLEGRO_SDL=OFF" "-DWANT_DEMO=OFF" "-DSHARED=0" "-DINSTALL_PKG_CONFIG_FILES=true" "-DWANT_EXAMPLES=OFF" "-DWANT_TESTS=OFF" "-DWANT_AUDIO=ON" "-DWANT_COLOR=ON" "-DWANT_D3D=ON" "-DWANT_D3D9EX=OFF" "-DWANT_DSOUND=ON" "-DWANT_FLAC=ON" "-DWANT_FONT=ON" "-DWANT_GLES3=ON" "-DWANT_IMAGE=ON" "-DWANT_IMAGE_JPG=ON" "-DWANT_IMAGE_PNG=ON" "-DWANT_MEMFILE=ON" "-DWANT_MODAUDIO=OFF" "-DWANT_NATIVE_DIALOG=ON" "-DWANT_NATIVE_IMAGE_LOADER=ON" "-DWANT_OGG_VIDEO=ON" "-DWANT_OPENAL=ON" "-DWANT_OPENGL=ON" "-DWANT_OPENSL=OFF" "-DWANT_OPUS=ON" "-DWANT_PHYSFS=ON" "-DWANT_POPUP_EXAMPLES=OFF" "-DWANT_PRIMITIVES=ON" "-DWANT_RELEASE_LOGGING=OFF" "-DWANT_SHADERS_D3D=ON" "-DWANT_SHADERS_GL=ON" "-DWANT_TREMOR=OFF" "-DWANT_TTF=ON" "-DWANT_VIDEO=ON" "-DWANT_VORBIS=ON" "-DCMAKE_MAKE_PROGRAM=/usr/bin/ninja" "-DCMAKE_SYSTEM_NAME=Linux" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/home/p_matveev/Program/VenGame/vcpkg/scripts/toolchains/linux.cmake" "-DVCPKG_TARGET_TRIPLET=x64-linux" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/home/p_matveev/Program/VenGame/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/home/p_matveev/Program/VenGame/vcpkg" "-D_VCPKG_INSTALLED_DIR=/home/p_matveev/Program/VenGame/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DWANT_ALLOW_SSE=ON"
CMake Deprecation Warning at CMakeLists.txt:8 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- [Allegro] Allegro Version: [5.2.9]
-- The C compiler identification is GNU 13.2.1
-- The CXX compiler identification is GNU 13.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Allowing GCC/Clang to use SSE instructions
-- Found PkgConfig: /usr/bin/pkg-config (found version "2.1.0") 
-- Looking for include file dirent.h
-- Looking for include file dirent.h - found
-- Looking for include file inttypes.h
-- Looking for include file inttypes.h - found
-- Looking for include file linux/input.h
-- Looking for include file linux/input.h - found
-- Looking for include file stdbool.h
-- Looking for include file stdbool.h - found
-- Looking for include file stdint.h
-- Looking for include file stdint.h - found
-- Looking for include file sys/io.h
-- Looking for include file sys/io.h - found
-- Looking for include file sys/stat.h
-- Looking for include file sys/stat.h - found
-- Looking for include file sys/time.h
-- Looking for include file sys/time.h - found
-- Looking for include file time.h
-- Looking for include file time.h - found
-- Looking for include file sys/utsname.h
-- Looking for include file sys/utsname.h - found
-- Looking for include file sys/types.h
-- Looking for include file sys/types.h - found
-- Looking for include file soundcard.h
-- Looking for include file soundcard.h - not found
-- Looking for include file sys/soundcard.h
-- Looking for include file sys/soundcard.h - found
-- Looking for include file machine/soundcard.h
...
Skipped 115 lines
...
-- Performing Test TTF_COMPILES
CMake Error at /home/p_matveev/Program/VenGame/vcpkg/buildtrees/allegro5/x64-linux-rel/CMakeFiles/CMakeScratch/TryCompile-BGJf4U/CMakeLists.txt:25 (target_link_libraries):
  Target "cmTC_aa945" links to:

    ZLIB::ZLIB

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



CMake Error at /usr/share/cmake/Modules/Internal/CheckSourceCompiles.cmake:101 (try_compile):
  Failed to generate test project build system.
Call Stack (most recent call first):
  /usr/share/cmake/Modules/CheckCSourceCompiles.cmake:52 (cmake_check_source_compiles)
  cmake/Common.cmake:28 (check_c_source_compiles)
  addons/CMakeLists.txt:105 (run_c_compile_test)


-- Configuring incomplete, errors occurred!
[2/2] "/usr/bin/cmake" -E chdir "../../x64-linux-dbg" "/usr/bin/cmake" "/home/p_matveev/Program/VenGame/vcpkg/buildtrees/allegro5/src/5.2.9.0-9723edc001.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=/home/p_matveev/Program/VenGame/vcpkg/packages/allegro5_x64-linux/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DCMAKE_PROJECT_INCLUDE=/home/p_matveev/Program/VenGame/vcpkg/ports/allegro5/cmake-project-include.cmake" "-DWANT_DOCS=OFF" "-DALLEGRO_SDL=OFF" "-DWANT_DEMO=OFF" "-DSHARED=0" "-DINSTALL_PKG_CONFIG_FILES=true" "-DWANT_EXAMPLES=OFF" "-DWANT_TESTS=OFF" "-DWANT_AUDIO=ON" "-DWANT_COLOR=ON" "-DWANT_D3D=ON" "-DWANT_D3D9EX=OFF" "-DWANT_DSOUND=ON" "-DWANT_FLAC=ON" "-DWANT_FONT=ON" "-DWANT_GLES3=ON" "-DWANT_IMAGE=ON" "-DWANT_IMAGE_JPG=ON" "-DWANT_IMAGE_PNG=ON" "-DWANT_MEMFILE=ON" "-DWANT_MODAUDIO=OFF" "-DWANT_NATIVE_DIALOG=ON" "-DWANT_NATIVE_IMAGE_LOADER=ON" "-DWANT_OGG_VIDEO=ON" "-DWANT_OPENAL=ON" "-DWANT_OPENGL=ON" "-DWANT_OPENSL=OFF" "-DWANT_OPUS=ON" "-DWANT_PHYSFS=ON" "-DWANT_POPUP_EXAMPLES=OFF" "-DWANT_PRIMITIVES=ON" "-DWANT_RELEASE_LOGGING=OFF" "-DWANT_SHADERS_D3D=ON" "-DWANT_SHADERS_GL=ON" "-DWANT_TREMOR=OFF" "-DWANT_TTF=ON" "-DWANT_VIDEO=ON" "-DWANT_VORBIS=ON" "-DCMAKE_MAKE_PROGRAM=/usr/bin/ninja" "-DCMAKE_SYSTEM_NAME=Linux" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/home/p_matveev/Program/VenGame/vcpkg/scripts/toolchains/linux.cmake" "-DVCPKG_TARGET_TRIPLET=x64-linux" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/home/p_matveev/Program/VenGame/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/home/p_matveev/Program/VenGame/vcpkg" "-D_VCPKG_INSTALLED_DIR=/home/p_matveev/Program/VenGame/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DWANT_ALLOW_SSE=OFF"
FAILED: ../../x64-linux-dbg/CMakeCache.txt 
"/usr/bin/cmake" -E chdir "../../x64-linux-dbg" "/usr/bin/cmake" "/home/p_matveev/Program/VenGame/vcpkg/buildtrees/allegro5/src/5.2.9.0-9723edc001.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=/home/p_matveev/Program/VenGame/vcpkg/packages/allegro5_x64-linux/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DCMAKE_PROJECT_INCLUDE=/home/p_matveev/Program/VenGame/vcpkg/ports/allegro5/cmake-project-include.cmake" "-DWANT_DOCS=OFF" "-DALLEGRO_SDL=OFF" "-DWANT_DEMO=OFF" "-DSHARED=0" "-DINSTALL_PKG_CONFIG_FILES=true" "-DWANT_EXAMPLES=OFF" "-DWANT_TESTS=OFF" "-DWANT_AUDIO=ON" "-DWANT_COLOR=ON" "-DWANT_D3D=ON" "-DWANT_D3D9EX=OFF" "-DWANT_DSOUND=ON" "-DWANT_FLAC=ON" "-DWANT_FONT=ON" "-DWANT_GLES3=ON" "-DWANT_IMAGE=ON" "-DWANT_IMAGE_JPG=ON" "-DWANT_IMAGE_PNG=ON" "-DWANT_MEMFILE=ON" "-DWANT_MODAUDIO=OFF" "-DWANT_NATIVE_DIALOG=ON" "-DWANT_NATIVE_IMAGE_LOADER=ON" "-DWANT_OGG_VIDEO=ON" "-DWANT_OPENAL=ON" "-DWANT_OPENGL=ON" "-DWANT_OPENSL=OFF" "-DWANT_OPUS=ON" "-DWANT_PHYSFS=ON" "-DWANT_POPUP_EXAMPLES=OFF" "-DWANT_PRIMITIVES=ON" "-DWANT_RELEASE_LOGGING=OFF" "-DWANT_SHADERS_D3D=ON" "-DWANT_SHADERS_GL=ON" "-DWANT_TREMOR=OFF" "-DWANT_TTF=ON" "-DWANT_VIDEO=ON" "-DWANT_VORBIS=ON" "-DCMAKE_MAKE_PROGRAM=/usr/bin/ninja" "-DCMAKE_SYSTEM_NAME=Linux" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/home/p_matveev/Program/VenGame/vcpkg/scripts/toolchains/linux.cmake" "-DVCPKG_TARGET_TRIPLET=x64-linux" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/home/p_matveev/Program/VenGame/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/home/p_matveev/Program/VenGame/vcpkg" "-D_VCPKG_INSTALLED_DIR=/home/p_matveev/Program/VenGame/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DWANT_ALLOW_SSE=OFF"
CMake Deprecation Warning at CMakeLists.txt:8 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- [Allegro] Allegro Version: [5.2.9]
-- The C compiler identification is GNU 13.2.1
-- The CXX compiler identification is GNU 13.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /usr/bin/pkg-config (found version "2.1.0") 
-- Looking for include file dirent.h
-- Looking for include file dirent.h - found
-- Looking for include file inttypes.h
-- Looking for include file inttypes.h - found
-- Looking for include file linux/input.h
-- Looking for include file linux/input.h - found
-- Looking for include file stdbool.h
-- Looking for include file stdbool.h - found
-- Looking for include file stdint.h
-- Looking for include file stdint.h - found
-- Looking for include file sys/io.h
-- Looking for include file sys/io.h - found
-- Looking for include file sys/stat.h
-- Looking for include file sys/stat.h - found
-- Looking for include file sys/time.h
-- Looking for include file sys/time.h - found
-- Looking for include file time.h
-- Looking for include file time.h - found
-- Looking for include file sys/utsname.h
-- Looking for include file sys/utsname.h - found
-- Looking for include file sys/types.h
-- Looking for include file sys/types.h - found
-- Looking for include file soundcard.h
-- Looking for include file soundcard.h - not found
-- Looking for include file sys/soundcard.h
-- Looking for include file sys/soundcard.h - found
-- Looking for include file machine/soundcard.h
-- Looking for include file machine/soundcard.h - not found
-- Looking for include file linux/soundcard.h
-- Looking for include file linux/soundcard.h - found
-- Looking for include file libkern/OSAtomic.h
-- Looking for include file libkern/OSAtomic.h - not found
-- Looking for include file sys/inotify.h
-- Looking for include file sys/inotify.h - found
-- Looking for include file sal.h
-- Looking for include file sal.h - not found
-- Looking for getexecname
-- Looking for getexecname - not found
-- Looking for mkstemp
-- Looking for mkstemp - found
-- Looking for mmap
-- Looking for mmap - found
-- Looking for mprotect
-- Looking for mprotect - found
-- Looking for sched_yield
-- Looking for sched_yield - found
-- Looking for sysconf
-- Looking for sysconf - found
-- Looking for fseeko
-- Looking for fseeko - found
-- Looking for ftello
-- Looking for ftello - found
-- Looking for strerror_r
-- Looking for strerror_r - found
-- Looking for strerror_s
-- Looking for strerror_s - not found
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of _Bool
-- Check size of _Bool - done
-- Performing Test ALLEGRO_HAVE_PROCFS_ARGCV
-- Performing Test ALLEGRO_HAVE_PROCFS_ARGCV - Failed
-- Performing Test ALLEGRO_HAVE_SV_PROCFS_H
-- Performing Test ALLEGRO_HAVE_SV_PROCFS_H - Failed
-- Performing Test ALLEGRO_HAVE_VA_COPY
-- Performing Test ALLEGRO_HAVE_VA_COPY - Success
-- Found OpenGL: /usr/lib/libOpenGL.so   
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Found ZLIB: optimized;/home/p_matveev/Program/VenGame/vcpkg/installed/x64-linux/lib/libz.a;debug;/home/p_matveev/Program/VenGame/vcpkg/installed/x64-linux/debug/lib/libz.a (found version "1.3.0")  
-- Found BZip2: optimized;/home/p_matveev/Program/VenGame/vcpkg/installed/x64-linux/lib/libbz2.a;debug;/home/p_matveev/Program/VenGame/vcpkg/installed/x64-linux/debug/lib/libbz2d.a (found version "1.0.8") 
-- Looking for BZ2_bzCompressInit
-- Looking for BZ2_bzCompressInit - found
-- Found PNG: optimized;/home/p_matveev/Program/VenGame/vcpkg/installed/x64-linux/lib/libpng16.a;debug;/home/p_matveev/Program/VenGame/vcpkg/installed/x64-linux/debug/lib/libpng16d.a (found version "1.6.40") 
-- Found X11: /usr/include   
-- Looking for XOpenDisplay in /usr/lib/libX11.so;/usr/lib/libXext.so
-- Looking for XOpenDisplay in /usr/lib/libX11.so;/usr/lib/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- Looking for XcursorImageCreate in Xcursor
-- Looking for XcursorImageCreate in Xcursor - found
-- Looking for X11/extensions/XInput2.h
-- Looking for X11/extensions/XInput2.h - found
-- Performing Test XINPUT2_COMPILES
-- Performing Test XINPUT2_COMPILES - Success
-- Looking for X11/extensions/xf86vmode.h
-- Looking for X11/extensions/xf86vmode.h - not found
-- Looking for XF86VidModeQueryExtension in Xxf86vm
-- Looking for XF86VidModeQueryExtension in Xxf86vm - found
-- Looking for X11/extensions/Xinerama.h
-- Looking for X11/extensions/Xinerama.h - found
-- Looking for XineramaQueryExtension in Xinerama
-- Looking for XineramaQueryExtension in Xinerama - found
-- Looking for X11/extensions/Xrandr.h
-- Looking for X11/extensions/Xrandr.h - found
-- Looking for XRRQueryExtension in Xrandr
-- Looking for XRRQueryExtension in Xrandr - found
-- Looking for X11/extensions/scrnsaver.h
-- Looking for X11/extensions/scrnsaver.h - found
-- Looking for XScreenSaverQueryExtension in Xss
-- Looking for XScreenSaverQueryExtension in Xss - found
-- Looking for XOpenIM in X11
-- Looking for XOpenIM in X11 - found
WARNING: FreeImage not found, disabling support
-- Found JPEG: /usr/lib/libjpeg.so (found version "80") 
-- Found WebP: /usr/include  
-- Performing Test OSS_COMPILES
-- Performing Test OSS_COMPILES - Success
-- Found OSS: 1  
-- Checking for module 'libpulse-simple'
--   Found libpulse-simple, version 16.1
-- Performing Test PULSEAUDIO_COMPILES
-- Performing Test PULSEAUDIO_COMPILES - Success
-- Checking for module 'alsa'
--   Found alsa, version 1.2.10
-- Found OpenAL: /home/p_matveev/Program/VenGame/vcpkg/installed/x64-linux/debug/lib/libopenal.a  
-- Found FLAC: /home/p_matveev/Program/VenGame/vcpkg/installed/x64-linux/include  
-- Performing Test FLAC_COMPILES
-- Performing Test FLAC_COMPILES - Success
-- Found OGG: /home/p_matveev/Program/VenGame/vcpkg/installed/x64-linux/include  
-- Found VORBIS: /home/p_matveev/Program/VenGame/vcpkg/installed/x64-linux/include  
-- Performing Test VORBIS_COMPILES
-- Performing Test VORBIS_COMPILES - Success
-- Found OPUS: /home/p_matveev/Program/VenGame/vcpkg/installed/x64-linux/include/opus  
-- Performing Test OPUS_COMPILES
-- Performing Test OPUS_COMPILES - Success
-- Could NOT find MiniMP3 (missing: MINIMP3_INCLUDE_DIRS) 
WARNING: minimp3 was not found
-- Found Freetype:  (found version "2.12.1")  
-- Performing Test TTF_COMPILES
CMake Error at /home/p_matveev/Program/VenGame/vcpkg/buildtrees/allegro5/x64-linux-dbg/CMakeFiles/CMakeScratch/TryCompile-eTQAjB/CMakeLists.txt:25 (target_link_libraries):
  Target "cmTC_f3421" links to:

    ZLIB::ZLIB

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



CMake Error at /usr/share/cmake/Modules/Internal/CheckSourceCompiles.cmake:101 (try_compile):
  Failed to generate test project build system.
Call Stack (most recent call first):
  /usr/share/cmake/Modules/CheckCSourceCompiles.cmake:52 (cmake_check_source_compiles)
  cmake/Common.cmake:28 (check_c_source_compiles)
  addons/CMakeLists.txt:105 (run_c_compile_test)


-- Configuring incomplete, errors occurred!
ninja: build stopped: subcommand failed.
```
</details>

--- a/ports/allegro5/cmake-project-include.cmake
+++ b/ports/allegro5/cmake-project-include.cmake
@@ -1,3 +1,8 @@
 if(MSVC AND CMAKE_SYSTEM_PROCESSOR STREQUAL "ARM64")
     add_compile_options(/Gy)
 endif()
+
+# https://gitlab.kitware.com/cmake/cmake/-/issues/25635
+if(CMAKE_VERSION VERSION_EQUAL "3.28.0" OR CMAKE_VERSION VERSION_EQUAL "3.28.1")
+    list(APPEND CMAKE_IGNORE_PATH "${VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/share/freetype")
+endif()

--- a/ports/allegro5/portfile.cmake
+++ b/ports/allegro5/portfile.cmake
@@ -13,10 +13,8 @@ if(VCPKG_TARGET_IS_ANDROID AND NOT ENV{ANDROID_HOME})
     message(FATAL_ERROR "${PORT} requires environment variable ANDROID_HOME to be set." )
 endif()
 
-if(NOT ENV{PKG_CONFIG})
-    vcpkg_find_acquire_program(PKGCONFIG)
-    set(ENV{PKG_CONFIG} "${PKGCONFIG}")
-endif()
+vcpkg_find_acquire_program(PKGCONFIG)
+set(ENV{PKG_CONFIG} "${PKGCONFIG}")
 
 string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" VCPKG_BUILD_SHARED_LIBS)
 

--- a/ports/allegro5/portfile.cmake
+++ b/ports/allegro5/portfile.cmake
@@ -9,61 +9,51 @@ vcpkg_from_github(
         msvc-arm64-atomic.patch
 )
 
+if(VCPKG_TARGET_IS_ANDROID AND NOT ENV{ANDROID_HOME})
+    message(FATAL_ERROR "${PORT} requires environment variable ANDROID_HOME to be set." )
+endif()
+
+if(NOT ENV{PKG_CONFIG})
+    vcpkg_find_acquire_program(PKGCONFIG)
+    set(ENV{PKG_CONFIG} "${PKGCONFIG}")
+endif()
 
 string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "dynamic" VCPKG_BUILD_SHARED_LIBS)
+
+vcpkg_check_features(OUT_FEATURE_OPTIONS options
+    FEATURES
+        direct3d    WANT_D3D
+        opengl      WANT_OPENGL
+)
 
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
         "-DCMAKE_PROJECT_INCLUDE=${CMAKE_CURRENT_LIST_DIR}/cmake-project-include.cmake"
-        -DWANT_DOCS=OFF
-        -DALLEGRO_SDL=OFF
-        -DWANT_DEMO=OFF
-        -DSHARED=${VCPKG_BUILD_SHARED_LIBS}
         -DINSTALL_PKG_CONFIG_FILES=true
-        -DWANT_EXAMPLES=OFF
-        -DWANT_TESTS=OFF
-        -DWANT_AUDIO=ON
-        -DWANT_COLOR=ON
-        -DWANT_D3D=ON
+        -DPKG_CONFIG_USE_CMAKE_PREFIX_PATH=ON
+        -DSHARED=${VCPKG_BUILD_SHARED_LIBS}
+        ${options}
+        -DALLEGRO_SDL=OFF
         -DWANT_D3D9EX=OFF # Not available on vcpkg
-        -DWANT_DSOUND=ON
-        -DWANT_FLAC=ON
-        -DWANT_FONT=ON
+        -DWANT_DEMO=OFF
+        -DWANT_DOCS=OFF
+        -DWANT_EXAMPLES=OFF
         -DWANT_GLES3=ON
-        -DWANT_IMAGE=ON
-        -DWANT_IMAGE_JPG=ON
-        -DWANT_IMAGE_PNG=ON
-        -DWANT_MEMFILE=ON
+        -DWANT_IMAGE_FREEIMAGE=OFF
         -DWANT_MODAUDIO=OFF # Not available on vcpkg right now
-        -DWANT_NATIVE_DIALOG=ON
-        -DWANT_NATIVE_IMAGE_LOADER=ON
-        -DWANT_OGG_VIDEO=ON
-        -DWANT_OPENAL=ON
-        -DWANT_OPENGL=ON
+        -DWANT_MP3=OFF
         -DWANT_OPENSL=OFF # Not yet available on vcpkg
-        -DWANT_OPUS=ON
-        -DWANT_PHYSFS=ON
         -DWANT_POPUP_EXAMPLES=OFF
-        -DWANT_PRIMITIVES=ON
-        -DWANT_RELEASE_LOGGING=OFF
-        -DWANT_SHADERS_D3D=ON
-        -DWANT_SHADERS_GL=ON
+        -DWANT_TESTS=OFF
         -DWANT_TREMOR=OFF # Not yet available on vcpkg
-        -DWANT_TTF=ON
-        -DWANT_VIDEO=ON
-        -DWANT_VORBIS=ON
-    OPTIONS_RELEASE
-        -DWANT_ALLOW_SSE=ON
-    OPTIONS_DEBUG
-        -DWANT_ALLOW_SSE=OFF
 )
 
 vcpkg_cmake_install()
 
 vcpkg_copy_pdbs()
 
-vcpkg_cmake_config_fixup(PACKAGE_NAME allegro CONFIG_PATH lib/cmake/allegro)
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/allegro)
 vcpkg_fixup_pkgconfig()
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share" "${CURRENT_PACKAGES_DIR}/debug/include")

--- a/ports/allegro5/portfile.cmake
+++ b/ports/allegro5/portfile.cmake
@@ -25,6 +25,9 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS options
         direct3d    WANT_D3D
         opengl      WANT_OPENGL
 )
+if(NOT WANT_OPENGL)
+    list(APPEND options -DWANT_X11=OFF)
+endif()
 
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"

--- a/ports/allegro5/portfile.cmake
+++ b/ports/allegro5/portfile.cmake
@@ -2,7 +2,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO liballeg/allegro5
     REF ${VERSION}
-    SHA512 2e7833b725718fdf11f1cd5999dc9fafca4b34adda15d3d8cb551c0eb00368d2e1c08b15f8beb766e2f208ffdc3b5e33175e969c7333ad6dc92f269522ac3d32
+    SHA512 5466e547a20bf22d606a385eeb9facc57b43c7f64689c724f82a572d4730dc62b2860829435b739a716ebca85fdc01c071f3e630048cdfd4799157e61fe815e9
     HEAD_REF master
     PATCHES
         do-not-copy-pdbs-to-lib.patch

--- a/ports/allegro5/vcpkg.json
+++ b/ports/allegro5/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "allegro5",
-  "version": "5.2.9.0",
+  "version": "5.2.9.1",
   "description": "Allegro is a cross-platform library mainly aimed at video game and multimedia programming. It handles common, low-level tasks such as creating windows, accepting user input, loading data, drawing images, playing sounds, etc. and generally abstracting away the underlying platform. However, Allegro is not a game engine: you are free to design and structure your program as you like.",
   "homepage": "https://github.com/liballeg/allegro5",
   "license": "BSD-3-Clause AND Zlib",

--- a/ports/allegro5/vcpkg.json
+++ b/ports/allegro5/vcpkg.json
@@ -2,18 +2,29 @@
   "name": "allegro5",
   "version": "5.2.9.1",
   "description": "Allegro is a cross-platform library mainly aimed at video game and multimedia programming. It handles common, low-level tasks such as creating windows, accepting user input, loading data, drawing images, playing sounds, etc. and generally abstracting away the underlying platform. However, Allegro is not a game engine: you are free to design and structure your program as you like.",
-  "homepage": "https://github.com/liballeg/allegro5",
+  "homepage": "https://liballeg.org/",
   "license": "BSD-3-Clause AND Zlib",
   "dependencies": [
+    {
+      "name": "alsa",
+      "platform": "linux"
+    },
     "bzip2",
-    "freetype",
+    {
+      "name": "freetype",
+      "default-features": false
+    },
     "libflac",
+    "libjpeg-turbo",
     "libogg",
     "libpng",
     "libtheora",
     "libvorbis",
+    {
+      "name": "libwebp",
+      "default-features": false
+    },
     "openal-soft",
-    "opengl",
     "opus",
     "opusfile",
     "physfs",
@@ -26,5 +37,28 @@
       "host": true
     },
     "zlib"
-  ]
+  ],
+  "default-features": [
+    {
+      "name": "direct3d",
+      "platform": "windows"
+    },
+    {
+      "name": "opengl",
+      "platform": "!(arm & windows) & !uwp & !xbox"
+    }
+  ],
+  "features": {
+    "direct3d": {
+      "description": "Direct3D support",
+      "supports": "windows"
+    },
+    "opengl": {
+      "description": "OpenGL support",
+      "supports": "!(arm & windows) & !uwp & !xbox",
+      "dependencies": [
+        "opengl"
+      ]
+    }
+  }
 }

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -42,14 +42,6 @@
 ace:arm-neon-android=fail
 ace:arm64-android=fail
 ace:x64-android=fail
-allegro5:arm-neon-android=fail
-allegro5:arm64-android=fail
-allegro5:x64-android=fail
-# Fails with "fatal error LNK1322: cannot avoid potential ARM hazard" even with /Gy
-allegro5:arm64-windows=fail
-# Fails to build in CMake 3.28 attempting to pass $<LINK_ONLY:ZLIB::ZLIB>; https://gitlab.kitware.com/cmake/cmake/-/issues/25635
-allegro5:x64-osx=fail
-allegro5:arm64-osx=fail
 ampl-asl:x64-android=fail
 apr:arm-neon-android=fail
 apr:arm64-android=fail

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -42,6 +42,10 @@
 ace:arm-neon-android=fail
 ace:arm64-android=fail
 ace:x64-android=fail
+# Needs SDK (in ENV{ANDROID_HOME})
+allegro5:arm-neon-android=fail
+allegro5:arm64-android=fail
+allegro5:x64-android=fail
 ampl-asl:x64-android=fail
 apr:arm-neon-android=fail
 apr:arm64-android=fail

--- a/versions/a-/allegro5.json
+++ b/versions/a-/allegro5.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "cf1dbe1a350897ce667b1ea08874f4a64411dc1f",
+      "git-tree": "1162ff1bdc43ad43036a30af2ca8214eeea566d9",
       "version": "5.2.9.1",
       "port-version": 0
     },

--- a/versions/a-/allegro5.json
+++ b/versions/a-/allegro5.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "d4ab4f2f967722e216ae4d59fa2deae74a1ccddc",
+      "version": "5.2.9.1",
+      "port-version": 0
+    },
+    {
       "git-tree": "93860d1ef432006526823ace61750eb6b0e86ec2",
       "version": "5.2.9.0",
       "port-version": 0

--- a/versions/a-/allegro5.json
+++ b/versions/a-/allegro5.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "bb9baaac1162dd0ab250e6e7f7e2f335fc812c57",
+      "git-tree": "cf1dbe1a350897ce667b1ea08874f4a64411dc1f",
       "version": "5.2.9.1",
       "port-version": 0
     },

--- a/versions/a-/allegro5.json
+++ b/versions/a-/allegro5.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "d4ab4f2f967722e216ae4d59fa2deae74a1ccddc",
+      "git-tree": "bb9baaac1162dd0ab250e6e7f7e2f335fc812c57",
       "version": "5.2.9.1",
       "port-version": 0
     },

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -81,7 +81,7 @@
       "port-version": 3
     },
     "allegro5": {
-      "baseline": "5.2.9.0",
+      "baseline": "5.2.9.1",
       "port-version": 0
     },
     "alpaca": {

